css/classify.styl

```diff
@@ -76,10 +76,10 @@
     transform: rotate(-15deg)
 
   > .task-area
-    flex: 1 1
+    // We want this fixed at 50h, but don't completely disable flex-shrink because
+    // when it wraps to its own line it shouldn't be cropped by small windows.
+    flex: 0 0.01 50ch
     margin-bottom: 1em
-    max-width: 60ch
```

Comment:
Looks good in chrome, but in firefox it's not shrinking to small windows. Perhaps there's a way to handle that with flexbox, otherwise adding something like this seemed to shrink it pretty well
```styl
    @media (max-width: 50ch)
      flex: none
      width: 100%
```